### ISSUE TITLE
Add IIIf icon to bottom corner of Mirador viewer

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme#1355]
- [ ] The DLME Mirador viewer should have the IIIf icon as in the [Stanford instance](https://exhibits.stanford.edu/parker/catalog/wz026zp2442).
- [ ] Should link to IIIf about page in English or Arabic depending on the language the user is in (dependent on https://github.com/sul-dlss/dlme/issues/1354)

**Comment by waynegraham** [comment]
I checked with IIIF folks and the closest page they currently have to the user-focused Stanford information is https://iiif.io/get-started/how-iiif-works/. We can perhaps coordinate with them to add a new page (if necessary) that is a bit more user-focused that explains IIIF and why it's awesome.

**Comment by mwerla** [comment]
@waynegraham Maybe the main IIIF page will be better? The "How IIIF works" is quite technical.

**Comment by jacobthill** [comment]
Another consideration is that we will have nothing in Arabic to link to so both views will link to the same English page.

**Comment by mwerla** [comment]
@jacobthill Good point, but I think that for now anyway if someone is interested in getting more about IIIF, English will be the main language. We are also linking to the IIIF website from the icon in the viewer, so maybe it's not that big problem. It would look worse if it would be linked for example from a "Find out more about IIIF" banner written in Arabic.

**Comment by thatbudakguy** [comment]
make it look exactly like exhibits. the icon isn't part of mirador.

**Comment by jcoyne** [comment]
I think this design may be confusing.  In other locations, that is a IIIF drag and drop icon, which you can use to pull the image into a IIIF viewer.   It is not a "Click here to find out more information about IIIF"

**Comment by thatbudakguy** [comment]
dropping this until we can come up with a different UI solution for giving people more info about IIIF, since the existing icon is used for a different purpsoe
